README.md

```diff
@@ -44,6 +44,7 @@ Please contribute. We are primarily interested in the following:
 
 After installing, please see the following areas to learn more:
 
+* [.NET IoT 101 (Jan 2020)](https://channel9.msdn.com/Series/IoT-101) - An introduction series how to create .NET IoT applications with a Raspberry Pi.
```

Comment:
```suggestion
* [.NET IoT 101 (Jan 2020)](https://channel9.msdn.com/Series/IoT-101) - An introduction series on how to create .NET IoT applications with a Raspberry Pi.
```

Reply:
I believe you still want the *on* word as I suggested, if you agree we can just commit that and merge. Thanks again @shaggygi

Reply:
Fixed wording.  Please review.